docs/index.rst

```diff
@@ -1,230 +1,232 @@
+5. The *map* task maps the input reads back to the final assembly using minimap2 to generate coverage data.
+6. The final *make_info_file* task produces a summary file documenting tool versions, parameters, memory usage, and Docker containers used throughout the workflow.
+
 
 Workflow Availability
 ---------------------
 
-The workflow from GitHub uses all the listed docker images to run all third-party tools.
-The workflow is available in GitHub: https://github.com/microbiomedata/metaAssembly; the corresponding Docker images are available in DockerHub: https://hub.docker.com/r/microbiomedata/spades and https://hub.docker.com/r/microbiomedata/bbtools
+The workflow from GitHub uses all the listed Docker images to run all third-party tools.  
+
+The workflow is available on GitHub: `https://github.com/microbiomedata/metaAssembly`  
+
+The corresponding Docker images are available on DockerHub:
+
+- `https://hub.docker.com/r/microbiomedata/spades`
+- `https://hub.docker.com/r/microbiomedata/bbtools`
 
 Requirements for Execution
 --------------------------
 
-(recommendations are in **bold**)  
+(Recommendations are in **bold**)  
 
 - WDL-capable Workflow Execution Tool (**Cromwell**)
 - Container Runtime that can load Docker images (**Docker v2.1.0.3 or higher**) 
 
 Hardware Requirements
 ---------------------
 
-- Memory: >40 GB RAM
+**Memory: >40 GB RAM**
 
 The memory requirement depends on the input complexity. Here is a simple estimation equation for the memory required based on kmers in the input file::
 
     predicted_mem = (kmers * 2.962e-08 + 1.630e+01) * 1.1 (in GB)
 
 .. note::
-    
-    The kmers variable for the equation above can be obtained using the kmercountmulti.sh script from BBTools.
 
-    kmercountmulti.sh -k=31 in=your.read.fq.gz
+   The kmers variable for the equation above can be obtained using the `kmercountmulti.sh` script from BBTools.
+
+   Example command:
 
+   ::
+
+       kmercountmulti.sh -k=31 in=your.read.fq.gz
 
 Workflow Dependencies
 ---------------------
 
-Third party software:  (This is included in the Docker image.)
+Third-party software: (This is included in the Docker image.)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- `metaSPades v3.15.0 <https://cab.spbu.ru/software/spades/>`_ (License: `GPLv2 <https://github.com/ablab/spades/blob/spades_3.15.0/assembler/GPLv2.txt>`_)
-- `BBTools:38.94 <https://jgi.doe.gov/data-and-tools/bbtools/>`_ (License: `BSD-3-Clause-LBNL <https://bitbucket.org/berkeleylab/jgi-bbtools/src/master/license.txt>`_)
+- `metaSPAdes v3.15.0 <https://cab.spbu.ru/software/spades/>`_ (License: `GPLv2 <https://github.com/ablab/spades/blob/spades_3.15.0/assembler/GPLv2.txt>`_)
+- `BBTools v38.94 <https://jgi.doe.gov/data-and-tools/bbtools/>`_ (License: `BSD-3-Clause-LBNL <https://bitbucket.org/berkeleylab/jgi-bbtools/src/master/license.txt>`_)
```

Comment:
v39.03
I think we should update line 33 and 43 of jgi_assembly.wdl  to v39.03 which JGI used. 
`bbtools_container = bbtools_container`